README.md

```diff
@@ -24,10 +24,12 @@ django-inline-actions adds actions to the InlineModelAdmin and ModelAdmin change
 ## Integration
 
 Add the `InlineActionsModelAdminMixin` to your `ModelAdmin`.
-If you want to have actions on your inlines, add the `InlineActionMixin` to your `InlineModelAdmin`.
-Each action is implemented as a method on the `ModelAdmin`/`InlineModelAdmin` and **must have** the following signature.
+If you want to have actions on your inlines, add the `InlineActionsMixin` to your `InlineModelsAdmin`.
+Each action is implemented as a method on the `ModelAdmin`/`InlineModelsAdmin` and **must have** the following signature.
```

Comment:
```suggestion
If you want to have actions on your inlines, add the `InlineActionsMixin` to your `InlineModelAdmin`.
Each action is implemented as a method on the `ModelAdmin`/`InlineModelAdmin` and **must have** the following signature.
```

Thanks for fixing the typo, but the inline admin mixin is still called [`InlineModel`](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#django.contrib.admin.InlineModelAdmin).